CellFabric/Cell_Fabric_FinFET__Mock/gen_lef.py

```diff
@@ -40,7 +40,7 @@ def s( x):
     fp.write( "  FOREIGN %s 0 0 ;\n" % out_lef)
 
     fp.write( "  SIZE %s BY %s ;\n" % ( s(j['bbox'][2]), s(j['bbox'][3])))
-    exclude_layers ={"via0","via1","via2","poly","LISD","SDT","fin","polycon","GCUT","active","nselect","pselect","nwell"}
+    exclude_layers ={"V0","V1","V2","poly","LISD","SDT","fin","polycon","GCUT","active","nselect","pselect","nwell"}
```

Comment:
Still need to generalize gen_lef & gen_gds_json instead of writing technology specific ones. Not urgent. Let's create an issue on this one.

Reply:
Created issue to come back and revisit this at a later point in time.